File: APPManage/AM.Model/xsweit_check.cs
/**  版本信息模板在安装目录下，可自行修改。
* xsweit_check.cs
*
* 功 能： N/A
* 类 名： xsweit_check
*
* Ver    变更日期             负责人  变更内容
* ───────────────────────────────────
* V0.01  2016/8/12 22:32:22   N/A    初版
*
* Copyright (c) 2012 Maticsoft Corporation. All rights reserved.
*┌──────────────────────────────────┐
*│　此技术信息为本公司机密信息，未经本公司书面同意禁止向第三方披露．　│
*│　版权所有：动软卓越（北京）科技有限公司　　　　　　　　　　　　　　│
*└──────────────────────────────────┘
*/
using System;
namespace AM.Model
{
	/// <summary>
	/// xsweit_check:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class xsweit_check
	{
		public xsweit_check()
		{}
		#region Model
		private string _guid;
		private string _userid;
		private DateTime? _createdate;
		private string _cpmc;
		private string _ggxh;
		private string _wtdw;
		private string _lxr;
		private string _scdw;
		private string _lxdh;
		private string _cpbh;
		private string _wtrq;
		private string _dwdz;
		private string _wtjyyq;
		private string _jyyj;
		private string _jyfy;
		private string _yqwcrq;
		private string _phwcxm;
		/// <summary>
		/// 
		/// </summary>
		public string Guid
		{
			set{ _guid=value;}
			get{return _guid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string userid
		{
			set{ _userid=value;}
			get{return _userid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? CreateDate
		{
			set{ _createdate=value;}
			get{return _createdate;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string CPMC
		{
			set{ _cpmc=value;}
			get{return _cpmc;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string GGXH
		{
			set{ _ggxh=value;}
			get{return _ggxh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string WTDW
		{
			set{ _wtdw=value;}
			get{return _wtdw;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string LXR
		{
			set{ _lxr=value;}
			get{return _lxr;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string SCDW
		{
			set{ _scdw=value;}
			get{return _scdw;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string LXDH
		{
			set{ _lxdh=value;}
			get{return _lxdh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string CPBH
		{
			set{ _cpbh=value;}
			get{return _cpbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string WTRQ
		{
			set{ _wtrq=value;}
			get{return _wtrq;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string DWDZ
		{
			set{ _dwdz=value;}
			get{return _dwdz;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string WTJYYQ
		{
			set{ _wtjyyq=value;}
			get{return _wtjyyq;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string JYYJ
		{
			set{ _jyyj=value;}
			get{return _jyyj;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string JYFY
		{
			set{ _jyfy=value;}
			get{return _jyfy;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string YQWCRQ
		{
			set{ _yqwcrq=value;}
			get{return _yqwcrq;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string PHWCXM
		{
			set{ _phwcxm=value;}
			get{return _phwcxm;}
		}
		#endregion Model

	}
}

File: AM/Scripts/appm/xsweit_check.js
function test() {
    var isok = false;
    if ($("#cpname").val() == "") {
        alert("产品名称不能为空");
        $("#cpname").focus();
    } else {
        if ($("#type").val() == "") {
            alert("规格型号不能为空");
            $("#type").focus();
        } else {
            if ($("#wtcompany").val() == "") {
                alert("委托单位不能为空");
                $("#wtcompany").focus();
            } else {
                if ($("#lxr").val() == "") {
                    alert("联系人不能为空");
                    $("#lxr").focus();
                } else {
                    if ($("#sccompany").val() == "") {
                        alert("生产单位不能为空");
                        $("#sccompany").focus();
                    } else {
                        if ($("#tel").val() == "") {
                            alert("联系电话不能为空");
                            $("#tel").focus();
                        } else {
                            if ($("#cpcode").val() == "") {
                                alert("产品编号不能为空");
                                $("#cpcode").focus();
                            } else {
                                if ($("#wtdate").val() == "") {
                                    alert("委托日期不能为空");
                                    $("#wtdate").focus();
                                } else {
                                    if ($("#address").val() == "") {
                                        alert("单位地址不能为空");
                                        $("#address").focus();
                                    } else {
                                        if ($("#wtcheck").val() == "") {
                                            alert("委托检验要求不能为空");
                                            $("#wtcheck").focus();
                                        } else {
                                            if ($("#checkyj").val() == "") {
                                                alert("检验依据不能为空");
                                                $("#checkyj").focus();
                                            } else {
                                                if ($("#checkmoney").val() == "") {
                                                    alert("检验费用不能为空");
                                                    $("#checkmoney").focus();
                                                } else {
                                                    if ($("#wcdate").val() == "") {
                                                        alert("要求完成日期不能为空");
                                                        $("#wcdate").focus();
                                                    } else {
                                                        if ($("#phproject").val() == "") {
                                                            alert("需委托单位配合完成的项目不能为空");
                                                            $("#phproject").focus();
                                                        } else
                                                            isok = true;
                                                    }
                                                }
                                            }
                                        }
                                    }
                                }
                            }
                        }
                    }
                }
            }
        }
    }
    return isok;
}

String.prototype.trim = function () {

    return this.replace(/(^\s*)|(\s*$)/g, '');
}

function submit() {
    var check = test();
    if (check)
    {
    var userid = $("#userid").val();
    var objArr = $("input[type=text]")
    var textareas = $("textarea");
  
    var json = {
        "xsweit_check": [{
            "Guid": "",
            "CreateDate": "",
            "userid": userid,
            "CPMC": $(objArr[0]).val(),
            "GGXH": $(objArr[1]).val(),
            "WTDW": $(objArr[2]).val(),
            "LXR": $(objArr[3]).val(),
            "SCDW": $(objArr[4]).val(),
            "LXDH": $(objArr[5]).val(),
            "CPBH": $(objArr[6]).val(),
            "WTRQ": $(objArr[7]).val(),
            "DWDZ": $(objArr[8]).val(),
            "WTJYYQ": $(textareas[0]).val(),
            "JYYJ": $(textareas[1]).val(),
            "JYFY": $(objArr[9]).val(),
            "YQWCRQ": $(objArr[10]).val(),
            "PHWCXM": $(textareas[2]).val()
        }]
    };
    $.ajax({
        type: "post",
        url: "/AppM/addxsweit_check",
        data: { obj: JSON.stringify(json) },
        success: function (data) {
            layer.msg("提交成功");
        }
    })
    
    }
}